exercises/say/say_test.exs

```diff
@@ -0,0 +1,90 @@
+if !System.get_env("EXERCISM_TEST_EXAMPLES") do
+  Code.load_file("say.exs", __DIR__)
+end
+
+ExUnit.start
+ExUnit.configure exclude: :pending, trace: true
+
+defmodule SayTest do
+  use ExUnit.Case
+
+  # @tag :pending
+  test "says zero in english" do
```

Comment:
@zporter can you update the test descriptions to match https://github.com/exercism/x-common/blob/master/exercises/say/canonical-data.json